lib/api.js

```diff
@@ -299,4 +301,43 @@ async function handleIssueComment(context, eventName, event) {
   }
 }
 
+async function handleRepositoryDispatch(context) {
```

Comment:
This looks like it's just copied from handleScheduleTrigger. Instead, there should be a common method, e.g. handleScheduleTriggerOrRepositoryDispatch

Reply:
That makes sense, I'll do it that way.  The logic behind copying/duplicating was that while the logic for ScheduleTrigger and RepositoryDispatch are the same right now, it's likely that in the future a feature/enhancement will apply to one and not the other